react-ui/src/components/FilterContainer/WhatContainer/WhatContainer.tsx

```diff
@@ -142,6 +190,14 @@ const WhatContainer = () => {
     }
   }, [filters.typeId]);
 
+  useEffect(() => {
+    if (filters.suitableFor.length) {
+      const suitableFor = filters.suitableFor;
+      setSuitableForState(stringToNumberArray(suitableFor));
+      setUseAgeFilter(true);
+    }
+  }, [useAgeFilter, filters.suitableFor.length]);
```

Comment:
Linter is giving a warning [react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920) for this effect. It should be handled one way or another for example by ignoring the line if it is working and it is a false positive warning.

Reply:
dependency fixed!